reports/tasks.py

```diff
@@ -76,6 +68,21 @@ def send_trial_instances_report(recipients=settings.TRIAL_INSTANCES_REPORT_RECIP
     end_of_last_month = (beginning_of_this_month - timedelta(days=1))
     beginning_of_last_month = end_of_last_month.replace(day=1)
 
+    # Get instances with an active betatestapplication,
+    # whose InstanceReference has an active OpenEdXAppServer
+    # and was created last month
+    instances = OpenEdXInstance.objects.exclude(
+        betatestapplication__isnull=True
+    ).filter(
+        ref_set__openedxappserver_set___is_active=True
+    ).filter(
+        ref_set__created__range=[
+            beginning_of_last_month.strftime("%Y-%m-%d"),
+            end_of_last_month.strftime("%Y-%m-%d")
+        ])
+    domains = [
+        instance.external_lms_domain or instance.internal_lms_domain for instance in instances
```

Comment:
@spokerman12, I think `instance.domain` will always contain the right value and so this `or` condition may not be needed.

Reply:
This part of the logic was already implemented, so I wouldn't want to rework it unnecessarily; plus, I don't think the command is tested thoroughly. As suggested before, I'd be more than happy to work on this on a different task.